tests/phpunit/CiviTest/CiviUnitTestCase.php

```diff
@@ -306,7 +308,7 @@ public static function setUpBeforeClass(): void {
       throw new \RuntimeException('CiviUnitTestCase requires CIVICRM_UF=UnitTests');
     }
 
-    \Civi\Test::asPreInstall([static::CLASS, 'buildEnvironment'])->apply(TRUE);
+    //\Civi\Test::asPreInstall([static::CLASS, 'buildEnvironment'])->apply(TRUE);
```

Comment:
I share the feeling that this line is excessive in how much it resets the database. However, instead of dropping the line, one should [remove the coercive TRUE](https://github.com/civicrm/civicrm-core/pull/31681/commits/ae50848c8bbb22685da5bc58ca4065edb33e67f3).